src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't use `*` selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings, or other styles with '*' is still inefficient for the browser.